modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache

```diff
@@ -121,10 +121,10 @@
     <swagger-core-version>1.5.8</swagger-core-version>
     <jetty-version>9.2.9.v20150224</jetty-version>
```

Comment:
For jetty, would you please upgrade it to use 9.2.15.v20160210 (the latest stable version for java7)?